dashboard/app/assets/javascripts/components/dialog.jsx

```diff
@@ -0,0 +1,53 @@
+window.dashboard = window.dashboard || {};
+
+window.dashboard.Dialog = (function (React) {
```

Comment:
Can we add some kind of document-level comment to this file explaining its purpose, since we expect lots of reuse?